recipes/_install-windows.rb

```diff
@@ -25,28 +25,48 @@
   end
 
 # If no version is specified, select the latest package
-dd_agent_msi = dd_agent_version ? "ddagent-cli-#{dd_agent_version}.msi" : 'ddagent-cli.msi'
-temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
+dd_agent_installer_basename = dd_agent_version ? "ddagent-cli-#{dd_agent_version}" : 'ddagent-cli'
```

Comment:
at this point the default installer URL is not different for the MSI (and the EXE has the same URL, except that it has an `.exe` extension).

In general though if we're concerned about the user experience of automatic upgrades failing when the default latest package is downloaded (shouldn't happen with this cookbook but could happen with user-maintained scripts) then we may consider using a different URL for the new installer.